playground/CosmosEndToEnd/CosmosEndToEnd.AppHost/Program.cs

```diff
@@ -10,4 +10,13 @@
 builder.AddProject<Projects.CosmosEndToEnd_ApiService>("api")
        .WithReference(db);
 
+// This project is only added in playground projects to support development/debugging
+// of the dashboard. It is not required in end developer code. Comment out this code
+// to test end developer dashboard launch experience. Refer to WorkloadAttributes.cs
+// for the path to the dashboard binary (defaults to a relative path to the artifacts
+// directory).
+builder.AddProject<Projects.Aspire_Dashboard>(KnownResourceNames.AspireDashboard)
+    .WithEnvironment("DOTNET_DASHBOARD_GRPC_ENDPOINT_URL", "http://localhost:5555")
```

Comment:
Can this be done in the app model, so that it can access the dashboard URL? That way it could still be dynamic, and is not needed in the `launchSettings.json`.

Reply:
I'll have a closer look at this tomorrow on another PR. I need to look at a few options to see if they are better.